src/__TESTS__/reducers.test.js

```diff
@@ -0,0 +1,60 @@
+import '@testing-library/jest-dom';
+import citiesReducer from '../redux/cities/cities';
+
+describe('Data reducer test', () => {
+  test('reducer has default state', () => {
+    const initialState = {
+      states: [
+        { state_name: 'loading states ...' },
+      ],
+      cities: ['init'],
+      data: {
+        labels: [],
+        datasets: [
+          {
+            label: 'Air Gases',
+            data: [],
+            fill: true,
+            backgroundColor: 'rgba(241, 197, 40, 0)',
+            borderColor: 'rgba(241, 197, 40, 1)',
+          },
+        ],
+      },
+      loading: true,
+      error: null,
+    };
+    const result = citiesReducer(undefined, { type: '' });
```

Comment:
## OPTIONAL 
* I strongly suggest you fix this error i get when i run your test, i mark this as optional because the previous CR missed this, But you can fix it in your free time to make your project awesome 💯  
![image](https://user-images.githubusercontent.com/52497246/218010941-46a1c991-d59e-495a-a13c-7addc35ee4a1.png)